src/select-field.jsx

```diff
@@ -141,6 +141,8 @@ const SelectField = React.createClass({
       hintText,
       fullWidth,
       errorText,
+      onFocus,
+      onBlur,
```

Comment:
Sure, will do. Looks like some of it was already documented when `SelectField` was split from `TextField` docs.